activiti-core/activiti-image-generator/src/main/java/org/activiti/image/impl/DefaultProcessDiagramGenerator.java

```diff
@@ -175,7 +182,13 @@ public void draw(DefaultProcessDiagramCanvas processDiagramCanvas,
                                                                       flowNode.getName(),
                                                                       graphicInfo,
                                                                       true);
+                    } else if(intermediateCatchEvent.getEventDefinitions().get(0) instanceof LinkEventDefinition) {
```

Comment:
You can reuse `isLinkCatchEvent` method:
```suggestion
                    } else if(intermediateCatchEvent.isLinkCatchEvent()) {
```